polkadot-protocol/smart-contract-basics/blocks-transactions-fees.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Blocks, Transactions and Fees for Asset Hub Smart Contracts
```

Comment:
This will for sure be too long. But let's send it and see what happens with it. Can revisit later if our reports yell at us about it haha